subworkflows/local/germline_variant_calling.nf

```diff
@@ -2,37 +2,12 @@
-include { BGZIP as BGZIP_VC_MANTA_SV                } from '../../modules/local/bgzip'
-include { BGZIP as BGZIP_VC_STRELKA                 } from '../../modules/local/bgzip'
-include { BGZIP as BGZIP_VC_STRELKA_GENOME          } from '../../modules/local/bgzip'
-include { CONCAT_VCF as CONCAT_DEEPVARIANT_GVCF     } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_DEEPVARIANT_VCF      } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_FREEBAYES            } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_HAPLOTYPECALLER      } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_MANTA_DIPLOID        } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_MANTA_SMALL_INDELS   } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_MANTA_SV             } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_STRELKA              } from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_STRELKA_GENOME       } from '../../modules/local/concat_vcf/main'
-include { DEEPVARIANT                               } from '../../modules/nf-core/modules/deepvariant/main'
-include { FREEBAYES                                 } from '../../modules/nf-core/modules/freebayes/main'
-include { GATK4_GENOTYPEGVCFS as GENOTYPEGVCFS      } from '../../modules/nf-core/modules/gatk4/genotypegvcfs/main'
-include { GATK4_HAPLOTYPECALLER as HAPLOTYPECALLER  } from '../../modules/nf-core/modules/gatk4/haplotypecaller/main'
-include { GATK_JOINT_GERMLINE_VARIANT_CALLING       } from '../../subworkflows/nf-core/gatk4/joint_germline_variant_calling/main'
-include { MANTA_GERMLINE                            } from '../../modules/local/manta/germline/main'
-include { STRELKA_GERMLINE                          } from '../../modules/nf-core/modules/strelka/germline/main'
-include { TABIX_BGZIPTABIX as TABIX_BGZIP_TIDDIT_SV } from '../../modules/nf-core/modules/tabix/bgziptabix/main'
-include { TABIX_TABIX as TABIX_VC_DEEPVARIANT_GVCF  } from '../../modules/nf-core/modules/tabix/tabix/main'
-include { TABIX_TABIX as TABIX_VC_DEEPVARIANT_VCF   } from '../../modules/nf-core/modules/tabix/tabix/main'
-include { TABIX_TABIX as TABIX_VC_FREEBAYES         } from '../../modules/nf-core/modules/tabix/tabix/main'
-include { TABIX_TABIX as TABIX_VC_HAPLOTYPECALLER   } from '../../modules/nf-core/modules/tabix/tabix/main'
-include { TIDDIT_SV                                 } from '../../modules/nf-core/modules/tiddit/sv/main'
+include { RUN_DEEPVARIANT     } from './variantcalling/deepvariant.nf'
+include { RUN_FREEBAYES       } from './variantcalling/freebayes.nf'
+include { RUN_HAPLOTYPECALLER } from './variantcalling/haplotypecaller.nf'
+include { RUN_MANTA_GERMLINE  } from './variantcalling/manta_germline.nf'
+include { RUN_STRELKA_SINGLE  } from './variantcalling/strelka_single.nf'
```

Comment:
What about `./variantcalling/<TOOL>/main` to be consistent with the other subworkflows?

Reply:
sure, than there are already a bit more adjusted to the nf-core style subworkflows